uint8 control
uint8 dig
uint8 central_drive
uint8 backhoe
uint8 bucket